scripts/configure_pyproject_toml.py

```diff
@@ -122,6 +122,20 @@ def parse_args():
         "scipy-openblas32>=0.3.26",
     ]
 
+    if backend == "lightning_gpu":
+        dependencies += ["custatevec-cu12"]
+
+    if backend == "lightning_tensor":
+        dependencies += ["cutensornet-cu12", "cutensor-cu12>=2.0.2", "nvidia-cusolver-cu12"]
```

Comment:
cutensor 2.0.2 is a [requirement](https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html) for cutensornet

Reply:
How could we ensure the version match across those libs here?

Reply:
I think we shouldn't pin the versions as long as they're `cu12`, and unless we know there are definitely broken old versions that do not work

Reply:
I asked this question because I'm not sure how nvidia libs sync the release with each other. That's say cutensor released a new version, while the latest cutensornet still depends on the previous one. Do we have to pin the versions to fix it or do you have any other options in mind to make it work?

Reply:
Actually `cutensor-cu12` is already a dependency of `cutensornet-cu12`:
```
$ pip install cutensornet-cu12
Collecting cutensornet-cu12
  Using cached cutensornet_cu12-2.6.0-py3-none-manylinux2014_aarch64.whl (2.2 MB)
Collecting cutensor-cu12<3,>=2.0.2
  Using cached cutensor_cu12-2.0.2-py3-none-manylinux2014_aarch64.whl (149.9 MB)
Installing collected packages: cutensor-cu12, cutensornet-cu12
Successfully installed cutensor-cu12-2.0.2 cutensornet-cu12-2.6.0
```

And this will take care of the `cutensor` version; perhaps we should remove `cutensor` all together?